vitest.config.js

```diff
@@ -0,0 +1,55 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+import { configDefaults, defineConfig } from 'vitest/config';
+
+const PODMAN_DESKTOP_EXCLUDED = [
+  '**/builtin/**',
+  '**/dist/**',
+  '**/.{cache,git,idea,output,temp,cdix}/**',
+  '**/*.cdix/**',
+];
+
+/**
+ * vitest workspace configuration for unit tests
+ */
+export default defineConfig({
+  test: {
```

Comment:
should it be there where we add the retry to 3 ?

Reply:
if we add it here, we retry 3 times for all the tests, where the previous config was only for the renderer and ui package